src/style.css

```diff
@@ -0,0 +1,46 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&subset=cyrillic");
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 40px;
+  padding: 10px 50px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+
+}
+
+.navigation > a {
+  position: relative;
+  margin-left: 20px;
```

Comment:
Are you sure you need left margin in the first element?

Reply:
No, but it requires another one css rule :)